simulation-scripts/perturb.sh

```diff
@@ -356,25 +244,23 @@ run_ssh() {
     root@"${@}"
 }
 
-get_file_to_run() {
-  local FILE="${1}"
-  echo "set -x"
-  if [[ "${IS_DRY_RUN:-}" == 1 ]]; then
-    cat "${FILE}" >&2
-  else
-    cat "${FILE}"
-  fi
-  echo "echo PERTURBANCE COMPLETE FOR ${FILE} ON \$(hostname) AT \$(date)"
-  echo
-}
-
 run_file_on_host() {
   local FILE="${1}"
   local HOST="${2}"
-  (
-    set -x
-    get_file_to_run "${FILE}"
-  ) | run_ssh "${HOST}"
+
+  scp \
```

Comment:
This change leaves code sat on the host that needs cleanup, would suggest these files go into `/tmp` and are cleaned up after a successful run.

Reply:
Thats makes sense,agreed

Reply:
Moved to /tmp and added cleanup to function